daprdocs/content/en/concepts/dapr-services/placement.md

```diff
@@ -14,3 +14,59 @@ The placement service Docker container is started automatically as part of [`dap
 ## Kubernetes mode
 
 The placement service is deployed as part of `dapr init -k`, or via the Dapr Helm charts. For more information on running Dapr on Kubernetes, visit the [Kubernetes hosting page]({{< ref kubernetes >}}).
+
+## Placement tables
+
+There is an HTTP API `/placement/state` for placement service that exposes the placement table. The API is exposed on the healthz server. Since this is an unauthenticated endpoint, so it is disabled by default, you need to set `DAPR_PLACEMENT_METADATA_ENABLED` environment or `metadata-enabled` command line args to true to enable it.
+
+### Usecase:
+The placement table API can be used for retrieving the current placement table, which contains all the actors registered. This can be helpful for debugging and also allow other tools (Dashboard) to extract and present for information about Actors.
+
+### HTTP Request
+
+```
+GET http://localhost:<healthzPort>/placement/state
+```
+
+### HTTP Response Codes
+
+Code | Description
+---- | -----------
+200  | Placement tables information returned
+500  | Placement could not return the placement tables information
+
+### HTTP Response Body
+
+**Placement tables API Response Object**
+
+Name                   | Type                                                                  | Description
+----                   | ----                                                                  | -----------
+tableVersion           | int                                                                   | The placement table version
+hostList               | [Actor Host Info](#actorhostinfo)[]                                   | A json array of registered actors host info.
+
+<a id="actorhostinfo"></a>**Actor Host Info**
+
+Name  | Type    | Description
+----  | ----    | -----------
+name  | string  | The host:port address of the actor.
+appId | string  | app id.
+entities | json string array | List of actor types it hosts.
+updatedAt | timestamp | Timestamp of the actor registered/updated.

```

Comment:
This is the timestamp that an actor connected to the placement and registered itself. For now, the actor type is unlikely to be changed in runtime, so this basically the time that the actor first connected to the placement and kept alive till now. This may be useful for users to know from which time the actor has kept an active heartbeat with placement.

Reply:
This applies to all actor instance? So if there are 2 actor types each with 500 actors, these are the heartbeats from all 1000 actor instances to the placement service? Hence I would expect this time to be updated very frequently, is that correct?

Reply:
No, this is not the latest heartbeat time.  If there are 2 actor types each with 500 actors, this would be the first time they connected to placement, and do not update if they are healthy. If there is one failed heartbeat(it would be removed from the list), and then back to connect to placement, the time would be updated.

Maybe changing the description to `Timestamp of the actor registered to placement` would be better?

Reply:
@Taction - Trying to close out this issue. 
1) If there are multiple actor types, are there multiple entries in the hostList, one for each actor type
List this
```
{
	"hostList": [{
		"name": "198.18.0.1:49347",
		"appId": "actor1",
		"actorTypes": ["testActorType1"],
		"updatedAt": 1690274322325260000
                },
                {
		"name": "198.18.0.1:49347",
		"appId": "actor2",
		"actorTypes": ["testActorType2"],
		"updatedAt": 1690274322325260000
	}],
	"tableVersion": 1
```

Also for the Update at this says "Timestamp" but it looks this in Linux time seconds. Can we make this time "updatedAt": 1690274322325260000"  a more friendly format?

Reply:
@Taction - Can you comment on this example and the timestamp

Reply:
Sorry for the late reply.

> If there are multiple actor types, are there multiple entries in the hostList, one for each actor type
List this

Every entry in the `hostList` represents one Dapr instance (which may hosts multiple actor types). `hostList` list all the Dapr instances that are connected to the placement service.
For example, if appId `actor1` hosts "testActorType1" and "testActorType3" actor types has one instance, the  appId `actor2` hosts "testActorType2" actor type has two instances with ip `198.18.0.2` and `198.18.0.3`:
```json
{
	"hostList": [{
		"name": "198.18.0.1:49347",
		"appId": "actor1",
		"actorTypes": ["testActorType1", "testActorType3"],
		"updatedAt": 1690274322325260000
                },
                {
		"name": "198.18.0.2:49347",
		"appId": "actor2",
		"actorTypes": ["testActorType2"],
		"updatedAt": 1690274322325260000
	        },
                {
		"name": "198.18.0.3:49347",
		"appId": "actor2",
		"actorTypes": ["testActorType2"],
		"updatedAt": 1690274322325260000
	}],
	"tableVersion": 1
```

> Also for the Update at this says "Timestamp" but it looks this in Linux time seconds. Can we make this time "updatedAt": 1690274322325260000" a more friendly format?

Yes, it is Unix Nanosecond time. I can send a pr to fix it to a more friendly format, RFC3339 ("2006-01-02T15:04:05Z07:00") or DateTime ( "2006-01-02 15:04:05") which would be better.

Reply:
@Taction - Can you include this example above into the docs at line 65, so that it show more than one actor in the list?

Reply:
Added